politeiawww/api/ticketvote/v1/v1.go

```diff
@@ -146,10 +146,13 @@ type Policy struct{}
 
 // PolicyReply is the reply to the Policy command.
 type PolicyReply struct {
-	LinkByPeriodMin int64  `json:"linkbyperiodmin"` // In seconds
-	LinkByPeriodMax int64  `json:"linkbyperiodmax"` // In seconds
-	VoteDurationMin uint32 `json:"votedurationmin"` // In blocks
-	VoteDurationMax uint32 `json:"votedurationmax"` // In blocks
+	LinkByPeriodMin        int64  `json:"linkbyperiodmin"` // In seconds
+	LinkByPeriodMax        int64  `json:"linkbyperiodmax"` // In seconds
+	VoteDurationMin        uint32 `json:"votedurationmin"` // In blocks
+	VoteDurationMax        uint32 `json:"votedurationmax"` // In blocks
+	SummariesPageSize      uint32 `json:"summariespagesize"`
+	InventoryPageSize      uint32 `json:"inventorypagesize"`
+	VoteTimestampsPageSize uint32 `json:"votetimestampspagesize"`
```

Comment:
The other APIs you use `TimestampsPageSize`, but here you use `VoteTimestampsPageSize`.  Shouldn't these be consistent?

Reply:
Oh I see.  I named the variable `VoteTimestampsPageSize`.  Dang, that's annoying.  These plugin politeiawww APIs will be removed at some point, so let's not carry the mistake over into the policy reply.  You should rename it to `TimestampsPageSize`.